packages/hash/external-services/docker-compose.yml

```diff
@@ -75,13 +75,39 @@ services:
     environment:
       HASH_GRAPH_PG_MIGRATION_URL: "postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${HASH_GRAPH_PG_DEV_DATABASE}"
 
+  telemetry-collector:
+    image: jaegertracing/all-in-one:1.40
+    deploy:
+      restart_policy:
+        condition: on-failure
+    healthcheck:
+      # Port 14269 is the Jaeger admin endpoint
+      test:
+        [
+          "CMD-SHELL",
+          "wget --no-verbose --tries=1 --spider http://localhost:14269 || exit 1",
+        ]
+      interval: 2s
+      timeout: 2s
+      retries: 10
```

Comment:
Are these the default values?

Reply:
They are not, I chose them to be similar to what we have for graph. It's a Go service, so it should boot quickly. Did you run into timeouts?

Reply:
No, I was just wondering 🙂